src/components/NetworkMap.vue

```diff
@@ -3,71 +3,137 @@
     <h3>Network Map</h3>
     <div class="map-container">
       <div class="wrapper">
-        <img ref="mapImage" src="/assets/mercator-map.png" alt="">
-        <canvas ref="mapCanvas"></canvas>
+        <img ref="mapImage" src="/assets/world-robinson.svg" alt="">
+        <svg ref="mapMarkers" id="mapMarkers"></svg>
       </div>
     </div>
   </div>
 </template>
 
 <script>
+// The network map uses a Robinson projection map image with slight cropping at the top and bottom of the map - https://simplemaps.com/resources/svg-world
+// co-ordinates are calculated with the following formulas:
+// x = 0.8474 RX (lng - lng0)
+// y = 1.3523 RY
+// - R is radius of globe at scale of the map, where equator length is 0.8487 of map width
+// - X and Y are coefficient values determined by Robinson at known latitudes, with intermediate values calculated by interpolation
+// (I have used linear interpolation for simplicity, at slight cost of accuracy)
+// the formula and coefficients can be found here: https://en.wikipedia.org/wiki/Robinson_projection#Formulation
+
+// known X and Y coefficients for latitudes at 5 degree intervals
+const latArray = [0, 5, 10, 15, 20, 25, 30, 35, 40, 45, 50, 55, 60, 65, 70, 75, 80, 85, 90]
+const XValues = [1, 0.9986, 0.9954, 0.9900, 0.9822, 0.9730, 0.9600, 0.9427, 0.9216, 0.8962, 0.8679, 0.8350, 0.7986, 0.7597, 0.7186, 0.6732, 0.6213, 0.5722, 0.5322]
+const YValues = [0, 0.0620, 0.1240, 0.1860, 0.2480, 0.3100, 0.3720, 0.4340, 0.4958, 0.5571, 0.6176, 0.6769, 0.7346, 0.7903, 0.8435, 0.8936, 0.9394, 0.9761, 1]
+
 export default {
   name: "NetworkMap",
-  data: function () {
+  data: function() {
     return {
-      mapLngLeft: -180,
-      mapLngRight: 180,
-      mapLatBottom: -57,
+      // known lontitude offset of map in degrees
+      lngOffset: -3
     }
   },
   props: ['points'],
-  computed: {
-    pointRadius() {
-      if (this.mapWidth > 450) return 5
-      if (this.mapWidth > 350) return 4
-      else return 3
-    },
-    mapHeight() {
-      return this.$refs.mapImage.clientHeight
-    },
-    mapWidth() {
-      return this.$refs.mapImage.clientWidth
-    }
-  },
   methods: {
-    convertGeoToXy(lat, lng) {
-      const mapLatBottomRad = this.mapLatBottom * Math.PI / 180
-      const latitudeRad = lat * Math.PI / 180
-      const mapLngDelta = (this.mapLngRight - this.mapLngLeft)
+    convertLatLngToXy(lat, lng, mapWidth, topOffset = 0, leftOffset = 0) {
+      // disallow invalid latitude or longtitude
+      if (Math.abs(lat) > 90 || Math.abs(lng) > 180) return
 
-      const worldMapWidth = ((this.mapWidth / mapLngDelta) * 360) / (2 * Math.PI)
-      const mapOffsetY = (worldMapWidth / 2 * Math.log((1 + Math.sin(mapLatBottomRad)) / (1 - Math.sin(mapLatBottomRad))))
+      // h and w are full height and width of the Robinson map without any cropping - the correct ratio is 1.97165551906973
+      const w = mapWidth
+      const h = w / 1.97165551906973
```

Comment:
Magic number detected! Give it an appropriate CONSTANT_NAME and use that instead.